dbsnp-importer/src/test/resources/application.properties

```diff
@@ -1,10 +1,25 @@
+dbsnp.datasource.driver-class-name=org.hsqldb.jdbcDriver
+dbsnp.datasource.url=jdbc:hsqldb:mem:db;sql.syntax_pgs=true;DB_CLOSE_DELAY=-1
+dbsnp.datasource.username=SA
+dbsnp.datasource.password=
+dbsnp.datasource.schema=src/test/resources/dbsnp-schema.sql
+dbsnp.datasource.data=src/test/resources/dbsnp-data.sql
```

Comment:
Is this renaming necessary to avoid the `spring.datasource` from loading those files into the job repository?

Reply:
yes, spring batch has problems when you don't want to put the jobRepository in your input DB. Having two datasources required non-trivial annotation sorcery. The approach taken was putting the jobRepository as autoconfigured (and unique) injected datasource, and handle the dbsnp datasource outside of spring.

I was really surprised that making this work was easier than the other way around.